QUARKUS-167.md

```diff
@@ -0,0 +1,39 @@
+# Quarkus-190 - Testing and verifying Quarkus/Vertx SQL
+
+The goal of this test plan is to cover the main points of conflicts between Quarkus/Vertx and SQL databases. 
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-190 
+
+
+## Scope of the testing
+Following actions were taken to ensure test coverage and automation for vertx/sql applications:
+ - Setup databases and load initial data through Flyway
+ - Programmatically insert statements
+ - Select statements
+ - Transactional statements between several tables
+ - Supported databases: postgresql, mysql/mariaDB, DB2
+
+### Impact on testsuites and testing automation:
+ - Ensure this coverage works both in JVM and NATIVE mode
+ - Integration test it's cover by Beefy-scenarios testsuite
+ - Some Openshift integration tests as Db2 are pending. 
```

Comment:
```suggestion
 - OpenShift integration tests with PosgreSQL
```

Reply:
Expecting reactive clients to be used and PG is the only supported one, the rest i mainly TP.
On traditional JDBC side we do not support DB2 drivers